src/MoBi.Presentation/Presenter/SelectReferenceAtParameterValuePresenter.cs

```diff
@@ -56,6 +57,30 @@ public override IEnumerable<ObjectBaseDTO> GetChildObjects(ObjectBaseDTO dto)
          return children;
       }
 
+      public override IReadOnlyList<ObjectPath> GetAllSelections()
+      {
+         var selectedItems = base.GetAllSelections();
+         replaceMoleculePropertiesWithModelParentName(selectedItems);
+         return selectedItems;
+      }
+
+      private void replaceMoleculePropertiesWithModelParentName(IReadOnlyList<ObjectPath> selectedItems)
+      {
+         var parentName = getParentNameFromSelection();
+         if (parentName != string.Empty)
+         {
+            foreach (var item in selectedItems.Where(x => x.PathAsString.Contains(Constants.MOLECULE_PROPERTIES)))
+            {
+               item.ReplaceWith(item.PathAsString.Replace(Constants.MOLECULE_PROPERTIES, parentName).ToPathArray());
+            }
+         }
+      }
+
+      private string getParentNameFromSelection() =>
+         _view.AllSelectedDTOs
+            .OfType<DummyParameterDTO>()
+            .FirstOrDefault()?.ModelParentName ?? string.Empty;
+
```

Comment:
it is possible to multiselect parameters from several nodes.
I think, in this case `.FirstOrDefault()` might return the wrong parent node (e.g. `Organism` in the example below) ??
![{DD3E9E05-27DD-45F0-8985-79F2F28BFE4F}](https://github.com/user-attachments/assets/562f399d-1624-4534-b8cb-bfa27ccf93b0)

Reply:
get parent name **for each selection separately**, no?
Otherwise if you multiselect 2 parameters from 2 different compounds, we will have the same problem (and now the same DTO type).